docs/scripts_md/TarchiveOB.md

```diff
@@ -0,0 +1,95 @@
+# NAME
+
+NeuroDB::objectBroker::TarchiveOB -- An object broker for tarchive records
```

Comment:
tarchive should be in code format or replace tarchive by "DICOM archive"

Reply:
Tarchive is now in code format.